modifytags/modifytags.go

```diff
@@ -0,0 +1,367 @@
+package modifytags
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/token"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+
+	"github.com/fatih/camelcase"
+	"github.com/fatih/structtag"
+)
+
+// A Mutation defines how the input is modified.
+type Mutation struct {
+	Remove        []string            // Remove tags for the comma separated list of keys.
+	RemoveOptions map[string][]string // Remove the list of options per given key.
+
+	Add                  []string            // Adds tags for the comma separated list of keys.
+	AddOptions           map[string][]string // Add the options per given key.
+	Override             bool                // Override current tags when adding tags.
```

Comment:
I wonder if 'Overwrite' would be a better name, with the comment `// if set, replace existing tags when adding`.

"Override" seems a bit inaccurate.